Makefile

```diff
@@ -1,6 +1,25 @@
 
 # Image URL to use all building/pushing image targets
 IMG ?= quay.io/konveyor/oadp-non-admin:latest
+
+SED = sed
+# if on macos, install gsed
+# https://formulae.brew.sh/formula/gnu-sed
+
+# Codecov OS String for use in download url
+ifeq ($(OS),Windows_NT)
+    OS_String = windows
```

Comment:
`OS_String` is used?

Reply:
It was used for codecov in oadp
https://github.com/openshift/oadp-operator/blob/14827fa839dce3ef68dbe63970588dfcf66d1aca/Makefile#L355
No not used here yet. Could be used later.